### ISSUE TITLE
Review all projects on the official HCA publications list and make sure they are in the project catalogue

### ISSUE THREAD

**Issue by gabsie** [ebi-ait/dcp-ingest-central#340]
Using [Dataset Tracking Sheet - Project Catalogue "Todo" Snapshot; Jun 3rd 2021](https://docs.google.com/spreadsheets/d/15imB5HErtY-wWaelnT_gxHRjWXyOQP6iagfPm0hU8sU/edit#gid=1934812232)

Check the list in group 3 tab matches the list at https://www.humancellatlas.org/publications/.
Check all projects in this group are in ingest. 
For any projects that are not already present, either add them or flag them for review (if they seem ineligible e.g. methods without new data)"

- [x] Check all projects from group 3 are in ingest and **flag any that are not**. Look them up by UUID and DOI

- [x] DO NOT DO IMPORT for the projects in this list, just produce a report for mismatches/misses
- [ ] Recommend how to fix.

**Comment by Wkt8** [comment]
I have checked. The list in Group 3 does not match up precisely to the list at https://www.humancellatlas.org/publications/ but I have included the new papers in my analysis as well. 

There were a number of benchmarking/review/meta-analysis studies in the official publication list that I included in the analysis and excluded from the below report to be added to the catalogue. 

Report here: 
There are a number of datasets in the snapshot but not in the project catalogue, and two datasets not in the snapshot, that are HCA publications, and that should be added to the dataset tracking sheet. 
https://docs.google.com/document/d/1DjKsLiWq8iCS1h_-x4ZNR6k95IjZvTVRe4uISQ2y5VI/edit
